### ISSUE TITLE
2025-01-08, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#45]
# Today's Arxiv Papers

(1) [Sa2VA: Marrying SAM2 with LLaVA for Dense Grounded Understanding of Images and Videos](http://arxiv.org/abs/2501.04001v1), 2025-01-07

(2) [RAG-Check: Evaluating Multimodal Retrieval Augmented Generation Performance](http://arxiv.org/abs/2501.03995v1), 2025-01-07

(3) [Influences on LLM Calibration: A Study of Response Agreement, Loss Functions, and Prompt Styles](http://arxiv.org/abs/2501.03991v1), 2025-01-07

(4) [(De)-Indexing and the Right to be Forgotten](http://arxiv.org/abs/2501.03989v1), 2025-01-07

(5) [VLM-driven Behavior Tree for Context-aware Task Planning](http://arxiv.org/abs/2501.03968v1), 2025-01-07

(6) [Vision Language Models as Values Detectors](http://arxiv.org/abs/2501.03957v1), 2025-01-07

(7) [Localizing AI: Evaluating Open-Weight Language Models for Languages of Baltic States](http://arxiv.org/abs/2501.03952v1), 2025-01-07

(8) [Not all tokens are created equal: Perplexity Attention Weighted Networks for AI generated text detection](http://arxiv.org/abs/2501.03940v1), 2025-01-07

(9) [Exploring the Potential of Large Language Models in Public Transportation: San Antonio Case Study](http://arxiv.org/abs/2501.03904v1), 2025-01-07

(10) [LLaVA-Mini: Efficient Image and Video Large Multimodal Models with One Vision Token](http://arxiv.org/abs/2501.03895v1), 2025-01-07

(11) [LEAP: LLM-powered End-to-end Automatic Library for Processing Social Science Queries on Unstructured Data](http://dx.doi.org/10.14778/3705829.3705843), 2025-01-07

(12) [AlphaPO -- Reward shape matters for LLM alignment](http://arxiv.org/abs/2501.03884v1), 2025-01-07

(13) [CL3DOR: Contrastive Learning for 3D Large Multimodal Models via Odds Ratio on High-Resolution Point Clouds](http://arxiv.org/abs/2501.03879v1), 2025-01-07

(14) [Progressive Document-level Text Simplification via Large Language Models](http://arxiv.org/abs/2501.03857v1), 2025-01-07

(15) [MedFocusCLIP : Improving few shot classification in medical datasets using pixel wise attention](http://arxiv.org/abs/2501.03839v1), 2025-01-07

(16) [KAnoCLIP: Zero-Shot Anomaly Detection through Knowledge-Driven Prompt Learning and Enhanced Cross-Modal Integration](http://arxiv.org/abs/2501.03786v1), 2025-01-07

(17) [Context-Alignment: Activating and Enhancing LLM Capabilities in Time Series](http://arxiv.org/abs/2501.03747v1), 2025-01-07

(18) [SLAM: Towards Efficient Multilingual Reasoning via Selective Language Alignment](http://arxiv.org/abs/2501.03681v1), 2025-01-07

(19) [SMIR: Efficient Synthetic Data Pipeline To Improve Multi-Image Reasoning](http://arxiv.org/abs/2501.03675v1), 2025-01-07

(20) [CommitShield: Tracking Vulnerability Introduction and Fix in Version Control Systems](http://arxiv.org/abs/2501.03626v1), 2025-01-07

(21) [LlaMADRS: Prompting Large Language Models for Interview-Based Depression Assessment](http://arxiv.org/abs/2501.03624v1), 2025-01-07

(22) [From Code to Compliance: Assessing ChatGPT's Utility in Designing an Accessible Webpage -- A Case Study](http://arxiv.org/abs/2501.03572v1), 2025-01-07

(23) [What Does a Software Engineer Look Like? Exploring Societal Stereotypes in LLMs](http://arxiv.org/abs/2501.03569v1), 2025-01-07

(24) [Applying Large Language Models in Knowledge Graph-based Enterprise Modeling: Challenges and Opportunities](http://arxiv.org/abs/2501.03566v1), 2025-01-07

(25) [Bridged Semantic Alignment for Zero-shot 3D Medical Image Diagnosis](http://arxiv.org/abs/2501.03565v1), 2025-01-07

(26) [Beyond Factual Accuracy: Evaluating Coverage of Diverse Factual Information in Long-form Text Generation](http://arxiv.org/abs/2501.03545v1), 2025-01-07

(27) [PromptGuard: Soft Prompt-Guided Unsafe Content Moderation for Text-to-Image Models](http://arxiv.org/abs/2501.03544v1), 2025-01-07

(28) [Deep Learning for Pathological Speech: A Survey](http://arxiv.org/abs/2501.03536v1), 2025-01-07

(29) [SenseRAG: Constructing Environmental Knowledge Bases with Proactive Querying for LLM-Based Autonomous Driving](http://arxiv.org/abs/2501.03535v1), 2025-01-07

(30) [A Sequential Optimal Learning Approach to Automated Prompt Engineering in Large Language Models](http://arxiv.org/abs/2501.03508v1), 2025-01-07

(31) [Can LLMs Design Good Questions Based on Context?](http://arxiv.org/abs/2501.03491v1), 2025-01-07

(32) [Entropy-Guided Attention for Private LLMs](http://arxiv.org/abs/2501.03489v1), 2025-01-07

(33) [Align-Pro: A Principled Approach to Prompt Optimization for LLM Alignment](http://arxiv.org/abs/2501.03486v1), 2025-01-07

(34) [Women, Infamous, and Exotic Beings: What Honorific Usages in Wikipedia Reveal about the Socio-Cultural Norms](http://arxiv.org/abs/2501.03479v1), 2025-01-07

(35) [Reading with Intent -- Neutralizing Intent](http://arxiv.org/abs/2501.03475v1), 2025-01-07

(36) [MTRAG: A Multi-Turn Conversational Benchmark for Evaluating Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2501.03468v1), 2025-01-07

(37) [ISSR: Iterative Selection with Self-Review for Vocabulary Test Distractor Generation](http://arxiv.org/abs/2501.03462v1), 2025-01-07

(38) [Activating Associative Disease-Aware Vision Token Memory for LLM-Based X-ray Report Generation](http://arxiv.org/abs/2501.03458v1), 2025-01-07

(39) [CoReQA: Uncovering Potentials of Language Models in Code Repository Question Answering](http://arxiv.org/abs/2501.03447v1), 2025-01-07

(40) [LLM4CVE: Enabling Iterative Automated Vulnerability Repair with Large Language Models](http://arxiv.org/abs/2501.03446v1), 2025-01-07

(41) [Finding A Voice: Evaluating African American Dialect Generation for Chatbot Technology](http://arxiv.org/abs/2501.03441v1), 2025-01-07

(42) [BoundingDocs: a Unified Dataset for Document Question Answering with Spatial Annotations](http://arxiv.org/abs/2501.03403v1), 2025-01-06

(43) [Existential Crisis: A Social Robot's Reason for Being](http://arxiv.org/abs/2501.03376v1), 2025-01-06

(44) [BoostStep: Boosting mathematical capability of Large Language Models via improved single-step reasoning](http://arxiv.org/abs/2501.03226v1), 2025-01-06

(45) [Dispider: Enabling Video LLMs with Active Real-Time Interaction via Disentangled Perception, Decision, and Reaction](http://arxiv.org/abs/2501.03218v1), 2025-01-06

(46) [Leveraging Explainable AI for LLM Text Attribution: Differentiating Human-Written and Multiple LLMs-Generated Text](http://arxiv.org/abs/2501.03212v1), 2025-01-06

(47) [Detecting AI-Generated Text in Educational Content: Leveraging Machine Learning and Explainable AI for Academic Integrity](http://arxiv.org/abs/2501.03203v1), 2025-01-06

(48) [The FACTS Grounding Leaderboard: Benchmarking LLMs' Ability to Ground Responses to Long-Form Input](http://arxiv.org/abs/2501.03200v1), 2025-01-06

(49) [CLIX: Cross-Lingual Explanations of Idiomatic Expressions](http://arxiv.org/abs/2501.03191v1), 2025-01-06

(50) [Semantic Captioning: Benchmark Dataset and Graph-Aware Few-Shot In-Context Learning for SQL2Text](http://arxiv.org/abs/2501.03166v1), 2025-01-06

(51) [Large language models for artificial general intelligence (AGI): A survey of foundational principles and approaches](http://arxiv.org/abs/2501.03151v1), 2025-01-06

(52) [VicSim: Enhancing Victim Simulation with Emotional and Linguistic Fidelity](http://arxiv.org/abs/2501.03139v1), 2025-01-06

(53) [PRMBench: A Fine-grained and Challenging Benchmark for Process-Level Reward Models](http://arxiv.org/abs/2501.03124v2), 2025-01-06

(54) [CAT: Content-Adaptive Image Tokenization](http://arxiv.org/abs/2501.03120v1), 2025-01-06

(55) [LangFair: A Python Package for Assessing Bias and Fairness in Large Language Model Use Cases](http://arxiv.org/abs/2501.03112v1), 2025-01-06

(56) [LiLMaps: Learnable Implicit Language Maps](http://arxiv.org/abs/2501.03304v1), 2025-01-06

(57) [Retrieval-Augmented TLAPS Proof Generation with Large Language Models](http://arxiv.org/abs/2501.03073v1), 2025-01-06

(58) [Design and implementation of tools to build an ontology of Security Requirements for Internet of Medical Things](http://arxiv.org/abs/2501.03067v1), 2025-01-06

(59) [ChronoSense: Exploring Temporal Understanding in Large Language Models with Time Intervals of Events](http://arxiv.org/abs/2501.03040v1), 2025-01-06

(60) [Quantization Meets Reasoning: Exploring LLM Low-Bit Quantization Degradation for Mathematical Reasoning](http://arxiv.org/abs/2501.03035v1), 2025-01-06

(61) [Analyzing Fine-tuning Representation Shift for Multimodal LLMs Steering alignment](http://arxiv.org/abs/2501.03012v1), 2025-01-06

(62) [CALM: Curiosity-Driven Auditing for Large Language Models](http://arxiv.org/abs/2501.02997v1), 2025-01-06

(63) [Registering Source Tokens to Target Language Spaces in Multilingual Neural Machine Translation](http://arxiv.org/abs/2501.02979v1), 2025-01-06

(64) [FlipedRAG: Black-Box Opinion Manipulation Attacks to Retrieval-Augmented Generation of Large Language Models](http://arxiv.org/abs/2501.02968v1), 2025-01-06

(65) [Socratic Questioning: Learn to Self-guide Multimodal Reasoning in the Wild](http://arxiv.org/abs/2501.02964v2), 2025-01-06

(66) [SceneVTG++: Controllable Multilingual Visual Text Generation in the Wild](http://arxiv.org/abs/2501.02962v2), 2025-01-06

(67) [MotionBench: Benchmarking and Improving Fine-grained Video Motion Understanding for Vision Language Models](http://arxiv.org/abs/2501.02955v1), 2025-01-06

(68) [A Soft Sensor Method with Uncertainty-Awareness and Self-Explanation Based on Large Language Models Enhanced by Domain Knowledge Retrieval](http://arxiv.org/abs/2501.03295v1), 2025-01-06

(69) [DeCon: Detecting Incorrect Assertions via Postconditions Generated by a Large Language Model](http://arxiv.org/abs/2501.02901v1), 2025-01-06

(70) [MDP3: A Training-free Approach for List-wise Frame Selection in Video-LLMs](http://arxiv.org/abs/2501.02885v1), 2025-01-06

(71) [IIMedGPT: Promoting Large Language Model Capabilities of Medical Tasks by Efficient Human Preference Alignment](http://arxiv.org/abs/2501.02869v1), 2025-01-06

(72) [Large Language Models for Video Surveillance Applications](http://arxiv.org/abs/2501.02850v1), 2025-01-06

(73) [Graph-based Retrieval Augmented Generation for Dynamic Few-shot Text Classification](http://arxiv.org/abs/2501.02844v1), 2025-01-06

(74) [Multi-Modal One-Shot Federated Ensemble Learning for Medical Data with Vision Large Language Model](http://arxiv.org/abs/2501.03292v1), 2025-01-06

(75) [ADePT: Adaptive Decomposed Prompt Tuning for Parameter-Efficient Fine-tuning](http://arxiv.org/abs/2501.03291v1), 2025-01-06

(76) [Randomly Sampled Language Reasoning Problems Reveal Limits of LLMs](http://arxiv.org/abs/2501.02825v1), 2025-01-06

(77) [InfiFusion: A Unified Framework for Enhanced Cross-Model Reasoning via LLM Fusion](http://arxiv.org/abs/2501.02795v1), 2025-01-06

(78) [CodeVision: Detecting LLM-Generated Code Using 2D Token Probability Maps and Vision Models](http://arxiv.org/abs/2501.03288v1), 2025-01-06

(79) [GeAR: Generation Augmented Retrieval](http://arxiv.org/abs/2501.02772v1), 2025-01-06

(80) [Visual Large Language Models for Generalized and Specialized Applications](http://arxiv.org/abs/2501.02765v1), 2025-01-06

(81) [Ultrasound-QBench: Can LLMs Aid in Quality Assessment of Ultrasound Imaging?](http://arxiv.org/abs/2501.02751v1), 2025-01-06

(82) [TARDiS : Text Augmentation for Refining Diversity and Separability](http://arxiv.org/abs/2501.02739v1), 2025-01-06

(83) [Artificial Intelligence in Creative Industries: Advances Prior to 2025](http://arxiv.org/abs/2501.02725v1), 2025-01-06

(84) [KG-CF: Knowledge Graph Completion with Context Filtering under the Guidance of Large Language Models](http://arxiv.org/abs/2501.02711v1), 2025-01-06

(85) [QuIM-RAG: Advancing Retrieval-Augmented Generation with Inverted Question Matching for Enhanced QA Performance](http://dx.doi.org/10.1109/ACCESS.2024.3513155), 2025-01-06

(86) [EAGLE: Enhanced Visual Grounding Minimizes Hallucinations in Instructional Multimodal Models](http://arxiv.org/abs/2501.02699v1), 2025-01-06
